.github/workflows/build.yml

```diff
@@ -73,13 +73,13 @@ jobs:
       - name: use kms backed certificates
         if: ${{ inputs.use_kms }}
         run: |
-          touch cert/gardenlinux-secureboot.db.arn
+          touch ./cert/gardenlinux-secureboot.db.arn
```

Comment:
does this change actually make a difference? this should be functionally identical, no?

Reply:
No, there's no functional difference here. Intended to improve readability here but after all it doesn't change much anyway.